frodo-kem/.gitignore

```diff
@@ -0,0 +1,27 @@
+# Generated by Cargo
+# will have compiled files and executables
+debug/
+target/
+
+# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
+# More information here https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html
+Cargo.lock
```

Comment:
This is a librarie so the lock file should be included according to the documentation comment. (There is a lock file in the workspace root)